### PR TITLE
Feature/comments

### PR DESCRIPTION
**Pull request by MaksymMartyniak**

### CODE REVIEW

File: SummerProject/article/views.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@transaction.atomic
+def comment_form(request, article_id):
+    article = get_object_or_404(Article, id=article_id)
+    if request.method == 'POST':
+        data = request.body.decode('utf8')
+        data = json.loads(data)
+        form = CommentForm(data)
+        if form.is_valid():
+            comment = form.save(commit=False)
+            # comment.content = data["content"]
```
Comment (AndriiStasiuk) [inline SummerProject/article/views.py]: Delete all comments

**Review by AndriiStasiuk — APPROVED** [review]

File: SummerProject/article/views.py (hunk 1 above)
Comment (MaksymMartyniak) [reply SummerProject/article/views.py]: Done

File: SummerProject/article/tests/test_comment_post.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        user = User.objects.get(email=email)
+
+        return user, password
+    return inner
+
+
+@pytest.fixture()
+def new_article(db, create_user):
+    new_user, _ = create_user()
+    print("new_user{}".format(new_user))
```
Comment (UlianaShevtsiv) [inline SummerProject/article/tests/test_comment_post.py]: Please remove `print`

File: SummerProject/article/tests/test_comment_post.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    return Article.objects.create(
+        headline="New one",
+        photo="https://oyebesmartest.com/public/uploads/preview/-11550470501p1ok6amcue.png",
+        video="https://www.youtube.com/watch?v=axsaC62UQOc",
```
Comment (UlianaShevtsiv) [inline SummerProject/article/tests/test_comment_post.py]: Please use some fake URLs here.

File: SummerProject/article/tests/test_comment_post.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+@pytest.mark.django_db
+def test_comment_post(client, new_article):
+    data = {
+        'content': "This is new comment"
+    }
+    response = client.post('/api/v1/articles/{}/comment/'.format(new_article.id),
+                           data=data,
+                           content_type='application/json')
+    comment = new_article.comments.get(id=1)
```
Comment (UlianaShevtsiv) [inline SummerProject/article/tests/test_comment_post.py]: The test will fail if created comment is created with any other id than `1`. Please change this to `.get(id=response['comment_id'])`

Comment (UlianaShevtsiv) [reply SummerProject/article/tests/test_comment_post.py]: If you do this change, you can get rid of the `assert` from line `50`

File: SummerProject/article/urls.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+""" Urls for articles """
+from django.urls import path
+from .views import comment_form, viewing_comments
+
+urlpatterns = [
+    path('<int:article_id>/comment/', comment_form),
```
Comment (UlianaShevtsiv) [inline SummerProject/article/urls.py]: Let's make this more RESTfull by creating new comment with `POST` to `<int:article_id>/comments/` and returning list of existing commends by `GET` to `<int:article_id>/comments/`

File: SummerProject/config/urls.py [hunk 6]
```diff
@@ -19,6 +19,7 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/v1/articles/', include('article.urls')),
```
Comment (UlianaShevtsiv) [inline SummerProject/config/urls.py]: Do you need this in scope of your MR?

**Review by UlianaShevtsiv — APPROVED** [review]
+1 after some comments are fixed

File: SummerProject/article/tests/test_comment_post.py (hunk 2 above)
Comment (MaksymMartyniak) [reply SummerProject/article/tests/test_comment_post.py]: Done

File: SummerProject/article/tests/test_comment_post.py (hunk 3 above)
Comment (MaksymMartyniak) [reply SummerProject/article/tests/test_comment_post.py]: Done

File: SummerProject/article/tests/test_comment_post.py (hunk 4 above)
Comment (MaksymMartyniak) [reply SummerProject/article/tests/test_comment_post.py]: Done

File: SummerProject/article/urls.py (hunk 5 above)
Comment (MaksymMartyniak) [reply SummerProject/article/urls.py]: Done

**Review by Liliia111 — APPROVED** [review]

**Review by filipenm — APPROVED** [review]

**Pull request merged**